SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/DynamicSubobjectsTest/DynamicSubobjectTestActor.cpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
For some reason this file wasn't picked up as a rename by git...
not a big deal though

Reply:
>:( git